frontend/ui/data/keyboardlayouts/en_HE_keyboard.lua

```diff
@@ -0,0 +1,152 @@
+local en_popup = require("ui/data/keyboardlayouts/keypopup/en_popup")
+local he_popup = require("ui/data/keyboardlayouts/keypopup/he_popup")
+local com = en_popup.com -- comma (,)
+local prd = en_popup.prd -- period (.)
+local _at = en_popup._at
+local _eq = en_popup._eq -- equals sign (=)
+local _A_ = en_popup._A_
+local _a_ = en_popup._a_
+local _B_ = en_popup._B_
+local _b_ = en_popup._b_
+local _C_ = en_popup._C_
+local _c_ = en_popup._c_
+local _D_ = en_popup._D_
+local _d_ = en_popup._d_
+local _E_ = en_popup._E_
+local _e_ = en_popup._e_
+local _F_ = en_popup._F_
+local _f_ = en_popup._f_
+local _G_ = en_popup._G_
+local _g_ = en_popup._g_
+local _H_ = en_popup._H_
+local _h_ = en_popup._h_
+local _I_ = en_popup._I_
+local _i_ = en_popup._i_
+local _J_ = en_popup._J_
+local _j_ = en_popup._j_
+local _K_ = en_popup._K_
+local _k_ = en_popup._k_
+local _L_ = en_popup._L_
+local _l_ = en_popup._l_
+local _M_ = en_popup._M_
+local _m_ = en_popup._m_
+local _N_ = en_popup._N_
+local _n_ = en_popup._n_
+local _O_ = en_popup._O_
+local _o_ = en_popup._o_
+local _P_ = en_popup._P_
+local _p_ = en_popup._p_
+local _Q_ = en_popup._Q_
+local _q_ = en_popup._q_
+local _R_ = en_popup._R_
+local _r_ = en_popup._r_
+local _S_ = en_popup._S_
+local _s_ = en_popup._s_
+local _T_ = en_popup._T_
+local _t_ = en_popup._t_
+local _U_ = en_popup._U_
+local _u_ = en_popup._u_
+local _V_ = en_popup._V_
+local _v_ = en_popup._v_
+local _W_ = en_popup._W_
+local _w_ = en_popup._w_
+local _X_ = en_popup._X_
+local _x_ = en_popup._x_
+local _Y_ = en_popup._Y_
+local _y_ = en_popup._y_
+local _Z_ = en_popup._Z_
+local _z_ = en_popup._z_
+local _א_ = he_popup._א_
```

Comment:
Is this variable name actually working? luacheck doesn't usually object without reason (see the failing CI at the bottom of the main PR page):

```
Luacheck results
Checking frontend/ui/data/keyboardlayouts/en_HE_keyboard.lua 1 error

    frontend/ui/data/keyboardlayouts/en_HE_keyboard.lua:59:8: expected statement near 'א'

Checking frontend/ui/data/keyboardlayouts/keypopup/he_popup.lua 1 error

    frontend/ui/data/keyboardlayouts/keypopup/he_popup.lua:2:6: expected '}' (to close '{' on line 1) near 'א'
```

Reply:
the variables work in the emulator, see the screenshots, i can rename all of the in ascii if needed it just will be more work.

Reply:
Indeed it seems that it works in LuaJIT, but it doesn't work in plain Lua. We'd better not introduce any needless incompatibility with regular Lua. :-)